core/templates/dev/head/pages/donate/donate.html

```diff
@@ -66,7 +66,7 @@ <h2 style="color: #fff; margin-top: 10px;">The Oppia Foundation</h2>
         <h2 style="color: #fff; font-size: 20px; margin-bottom: 20px; margin-top: 20px;">
           Your generous gift funds:</h2>
         <div style="margin-bottom: 16px; text-align: center;">
-          <img ng-src="<[donateImgUrl]>">
+          <img ng-src="<[donateImgUrl]>" alt = "Funds">
```

Comment:
No spaces between `alt` and `=` and `"Funds"`. Change text to "Server costs, student outreach, marketing"